### ISSUE TITLE
Missing alt text for background image

### ISSUE THREAD

**Issue by elsa-ng** [elsa-ng/sinoalice#6]
Sometimes there are dependencies in the CSS stack and messing with it can cause upset the design and layout of the site, or a request to change the code could get hung up in approval from various stakeholders. In cases where the author has to have the background image in the <div> that wraps up other content, then a hacky fallback is to do this.
```
<div class="background-image" >
<span role="img" aria-label="[place alt text here]> </span>
[all the rest of my content]
</div>
```

This is a hack because semantically the alternate text is not on the element that actually has the image. However, from a screen reader perspective the div with the background image is ignored and so placing the span directly following it will provide that information in a way that will seem as if the alternate text was in the same place as the background image.

Summary
Try not to use CSS for important informational images
For ambient images that are CSS, it is a courtesy to provide alternate text. When doing so, place image in its own empty span with an aria-label and role="img.This is also true, in a situation where CSS must be used for information content.
If the div with the CSS image absolutely MUST contain other content, then provide an empty span with an aria-label and role="img" immediately following the <div> that has the image.
